src/wdio/services/TerraToolkitServeStaticService.js

```diff
@@ -0,0 +1,44 @@
+import serve from '../../../scripts/serve/serve-static';
+
+export default class TerraToolkitServeStaticService {
+  async onPrepare(config) {
+    if (!config.webpackConfig) {
+      // eslint-disable-next-line no-console
+      console.log('[Terra-Toolkit:serve-static] No webpack configuration provided');
```

Comment:
Should we log this as a warning?

Reply:
Should do we want to follow the other service logging 'convention' (is it?) with `console.log([ServeStaticService])`

Also, should it be StaticServe instead of ServeStatic?

Reply:
I can update to a warning.
I don't think there is a convention and I wanted consumers to know where this service was coming from so there is no confusion that it could be a third party service and I don't want to reverse serve static.